docs/farming.md

```diff
@@ -98,17 +78,7 @@ The address of your account will be necessary at the last step.
 # Replace `NODE_FILE_NAME` with the name of the node file you downloaded from releases
 # Replace `INSERT_YOUR_ID` with a nickname you choose
 # Copy all of the lines below, they are all part of the same command
-./NODE_FILE_NAME \
-  --chain testnet \
-  --wasm-execution compiled \
-  --execution wasm \
-  --bootnodes "/dns/farm-rpc.subspace.network/tcp/30333/p2p/12D3KooWPjMZuSYj35ehced2MTJFf95upwpHKgKUrFRfHwohzJXr" \
-  --rpc-cors all \
-  --rpc-methods unsafe \
-  --ws-external \
-  --validator \
-  --telemetry-url "wss://telemetry.polkadot.io/submit/ 1" \
-  --name INSERT_YOUR_ID
+./NODE_FILE_NAME --chain testnet --wasm-execution compiled --execution wasm --bootnodes "/dns/farm-rpc.subspace.network/tcp/30333/p2p/12D3KooWPjMZuSYj35ehced2MTJFf95upwpHKgKUrFRfHwohzJXr" --rpc-cors all --rpc-methods unsafe --ws-external --validator --telemetry-url "wss://telemetry.polkadot.io/submit/ 1" --name INSERT_YOUR_ID
```

Comment:
Why are these in one line again?

Reply:
I remembered we discussed a while ago that it would be best to consolidate to a more native command. It also helped with the readability of the overall document since each set was not hidden be details tab any more.

Reply:
I'm not sure that it helps readability when line is 327 characters long and needs you to update some values in it.